pkg/controllers/status/common.go

```diff
@@ -21,6 +21,15 @@ import (
 	"github.com/karmada-io/karmada/pkg/util/restmapper"
 )
 
+// Fake predicate for inheriting concurrency of `forType` from global options.
+// Matching the feature in later version.
+var bindingPredicateFn = builder.WithPredicates(predicate.Funcs{
```

Comment:
I think this name can be set more commonly, how about `interceptAllPredicateFn` or something else?